resources/sass/_map.scss

```diff
@@ -1,3 +1,42 @@
 .leaflet-tile-pane > .leaflet-layer {
   mix-blend-mode: multiply;
 }
+
+.legend {
+  background-color: $white;
+  color: $gray-text;
+  padding: 15px 0 10px 20px;
+  text-transform: uppercase;
+  width: 200px;
+
+  .title {
+    font-size: 11px;
+    margin-bottom: 10px;
+  }
+
+  .color-scale {
+    font-size: 14px;
+
+    div {
+      display: inline-block;
+    }
+
+    .color-bar {
+      background-color: #DCDFD7;
+      height: 15px;
+      width: 60%;
+      margin: 0 10px 0 10px;
+      position: relative;
+      top: 2px;
+
+      :first-child {
+        height: 100%;
+        width: 100%;
+        opacity: 0.5;
+
+        // this gradient should be kept in sync with mapserver's class definitions.
+        background: linear-gradient(to right, #FFC800 0%,#FF9600 8%,#FF6400 20%,#FF3200 39%,#FF1400 59%,#FF0000 100%);
```

Comment:
I don't like that this being data is saved as a static style in a CSS file. Can't this be saved in the database and injected via properties from the reagent component?